components/Graph.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { LineChart, Line, CartesianGrid, XAxis, YAxis, Label } from 'recharts';
+
+function Graph(props) {
+  return <div id="graph">
+    <LineChart width={800} height={400} data={props.data}>
+    <Line type="monotone" dataKey="primary_google_trends_vol" stroke="#8884d8" />
```

Comment:
Indentation needed from line 7 to 13.
Also, it's normally good practice to wrap your jsx code in parenthesis, for example:
return (\<div id="graph"\>
  ...
  ...
  ...
\</div\>);